win-odbc64/odbcclient/drvr35/cdesc.cpp

```diff
@@ -625,7 +645,6 @@ unsigned long CDescRec::setDescRec(short DescMode, SQLItemDesc_def *SQLItemDesc)
 	case SQL_LONGVARCHAR:
 	case SQL_WCHAR:
 	case SQL_WVARCHAR:
-		m_DescPrecision = m_DescLength;

```

Comment:
The location of local desciptor value assigning is adjusted. For other column type, param _m_DescPrecision_ is assigned between line 346 and 508, while previously for SQL_CHAR/SQL_VARCHAR/SQL_LONGVARCHAR/SQL_WCHAR/SQL_WVARCHAR columns the param is assigned in another _switch..case_ block. Moreover, param _m_ODBCPrecision_ is assigned in line 509 by param _m_DescPrecision_ and will be passed to function _ConvertCToSQL_ afterwards in this issue. As you see, the assigning of _m_DescPrecision_ is put ahead. This change is based on consideration of correct value passing and unified code style.